site/gatsby-site/src/utils/AlgoliaUpdater.js

```diff
@@ -45,7 +47,9 @@ const includedCSETAttributes = [
 ];
 
 const getClassificationArray = ({ classification, taxonomy }) => {
```

Comment:
please consider renaming, especially now that an object is being returned. Even better would be providing an comment example with the return type, considering the readability of `list` and the return object.